src/ragas/metrics/__init__.py

```diff
@@ -1,4 +1,11 @@
-from ragas.metrics._answer_correctness import AnswerCorrectness, answer_correctness
+from ragas.metrics._answer_accuracy import (
```

Comment:
I think we should change the name of the module from where we are importing this. Class name should not have a `_` in between them